app/views/help/cookie_settings.html.erb

```diff
@@ -28,27 +28,26 @@
   <div class="cookie-settings__no-js">
     <%= render "govuk_publishing_components/components/govspeak", {
       } do %>
-      <p>We use Javascript to set most of our cookies. Unfortunately Javascript is not running on your browser, so you cannot change your settings. You can try:</p>
+      <p><%= t('cookies.javascript') %></p>
       <ul>
-        <li>reloading the page</li>
-        <li>turning on Javascript in your browser</li>
+        <li><%= t('cookies.reload') %></li>
```

Comment:
Possibly the same as the previous list, if you decide to implement this as a loop.